src/Fixer/Casing/NativeFunctionTypeDeclarationCasingFixer.php

```diff
@@ -68,8 +72,15 @@ public function __construct()
         ];
 
         if (\PHP_VERSION_ID >= 80000) {
-            $this->hints = array_merge($this->hints, ['static' => true]);
-            $this->hints = array_merge($this->hints, ['mixed' => true]);
+            $this->hints = array_merge(
```

Comment:
Can't it simply be `$this->hints['false'] = true;` and so on?

Reply:
Yeah, probably it would be better 👍 I can change it in my PR or wait for @SpacePossum and change it only for `true` which I added there.